contrib/win32/install/openssh.wixproj

```diff
@@ -24,6 +25,12 @@
         </DefineConstants>
     </PropertyGroup>
 
+    <PropertyGroup Condition="'$(Platform)' == 'ARM64'">
+        <!-- Assumes WixToolPath input is provided at runtime -->
```

Comment:
Maybe add a reference to this doc in the comments just so we don't lose where the guidance comes from? Or maybe just mention in the PR description. https://wixtoolset.org/docs/v3/msbuild/daily_builds/